src/main/java/address/controller/MainController.java

```diff
@@ -60,7 +61,7 @@ public MainController(MainApp mainApp, ModelManager modelManager, Config config)
 
     public void start(Stage primaryStage) {
         this.primaryStage = primaryStage;
-        setTitle(config.appTitle, PreferencesManager.getInstance().getPersonFile());
+        primaryStage.setTitle(config.appTitle);
```

Comment:
Just confirming: we don't want to show filename in the title? It might be a useful info.